custom/icds_reports/tasks.py

```diff
@@ -172,7 +172,7 @@ def aggregate_tables(self, current_task, future_tasks):
 
 @periodic_task(
     queue='background_queue',
-    run_every=crontab(day_of_week='sunday,wednesday', minute=0, hour=21),
+    run_every=crontab(day_of_week='tuesday,thursday,saturday', minute=0, hour=21),
```

Comment:
There's something weird about this task. I haven't pinpointed it yet, but I think it's pulling in way more cases than it should

Reply:
good to know. let me know if you want a second pair of :eyes: